holidays/countries/aland_islands.py

```
@@ -0,0 +1,63 @@
+#  holidays
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)